jax/experimental/jax2tf/jax2tf.py

```diff
@@ -809,6 +810,79 @@ def _argminmax(fn, operand, axes, index_dtype):
 tf_impl[lax.cumsum_p] = tf.math.cumsum
 tf_impl[lax.cumprod_p] = tf.math.cumprod
 
+def _select_and_gather_add(tangents: TfVal,
+                           operand: TfVal,
+                           select_prim: core.Primitive,
+                           window_dimensions: Sequence[int],
+                           window_strides: Sequence[int],
+                           base_dilation: Sequence[int],
+                           window_dilation: Sequence[int],
+                           padding: Sequence[Tuple[int, int]],
+                           max_bits=64):
+  dtype = to_jax_dtype(operand.dtype)
```

Comment:
I think that a comment saying that this follows the translation rule would be useful.

Reply:
Addressed in `6b1a6ab`